pkg/flavor/kubernetes/nfs.go

```diff
@@ -179,32 +186,15 @@ func (flavor *Flavor) createServiceAccount(nfsNamespace string) error {
 	return nil
 }
 
-func (flavor *Flavor) createNFSConfigMap(nfsNamespace string) error {
-	log.Tracef(">>>>> createNFSConfigMap with namespace %s", nfsNamespace)
+func (flavor *Flavor) createNFSConfigMap(nfsNamespace, hostDomain string) error {
+	log.Tracef(">>>>> createNFSConfigMap with namespace %s, domain %s", nfsNamespace, hostDomain)
 	defer log.Tracef("<<<<< createNFSConfigMap")
 
-	nfsGaneshaConfig := `
-NFS_Core_Param
-{
-  NFS_Protocols= 4;
-  NFS_Port = 2049;
-  fsid_device = false;
-}
-
-EXPORT
-{
-  Export_Id = 716;
-  Path = /export;
-  Pseudo = /export;
-  Access_Type = RW;
-  Squash = No_Root_Squash;
-  Transports = TCP;
-  Protocols = 4;
-  SecType = "sys";
-  FSAL {
-	  Name = VFS;
-  }
-}`
+	// Format for pretty display with configmap
+	nfsCoreParamBlock := "NFS_Core_Param" + "\n" + "{" + "\n" + "NFS_Protocols= 4;" + "\n" + "NFS_Port = 2049;" + "\n" + "fsid_device = false;" + "\n" + "}" + "\n"
+	nfs4Block := "NFSv4" + "\n" + "{" + "\n" + "Graceless = true;" + "\n" + "UseGetpwnam = true;" + "\n" + "DomainName = " + hostDomain + ";" + "\n" + "}" + "\n"
+	exportBlock := "EXPORT" + "\n" + "{" + "\n" + "Export_Id = 716;" + "\n" + "Path = /export;" + "\n" + "Pseudo = /export;" + "\n" + "Access_Type = RW;" + "\n" + "Squash = No_Root_Squash;" + "\n" + "Transports = TCP;" + "\n" + "Protocols = 4;" + "\n" + "SecType = \"sys\";" + "\n" + "FSAL" + "\n" + "{" + "\n" + "Name = VFS;" + "\n" + "}" + "\n" + "}" + "\n"
+	nfsGaneshaConfig := nfsCoreParamBlock + nfs4Block + exportBlock
```

Comment:
It looks prettier the way it was... at least in code :)

Reply:
i had an extra tab in the original code looks like, so during edit of configmap using kubectl, it was escaping all new lines. Now its fixed, so reverted back.